content/AlgorytmyWykladniczeZadania.tex

```diff
@@ -89,36 +89,29 @@ \subsection{Zadanie 3 -- Kolorowanie grafu z list o rozmiarze nie większym niż
 
 \paragraph{Rozwiązanie:} 
 
-Niech graf $G$ to graf wejściowy w naszym algorytmie, taki, że
+Niech graf $G$ to graf wejściowy w naszym algorytmie taki, że
 $V(G) = \{v_1, v_2, \ldots, v_n\}$. 
-Niech $\chi = \bigcup_{i = 1}^n L_{v_i}$.
 Utwórzmy zbiór wierzchołków
-\[V' = \{v_{ij} : j \in L_{v_i} \land i \in [n] \land j \in \chi\},\]
+\[V' = \{v_{ik} : v_i\in V(G) \land k \in L_{v_i}\},\]
 
 oraz zbiór krawędzi 
-\[E' = \{v_{ij}v_{kl} : i = k \land i,k \in [n] \land j, l \in \chi\} \cup 
-\]
-\[\cup
-\{v_{ij}v_{kl} : j = l \land v_iv_k \in E(G) \land i,k \in [n] \land j, l \in \chi\}.\]
+\[E' = \underbrace{\{v_{ik}v_{il} : v_i \in V(G) \land k, l \in L_{v_i}\}}_\text{krawędzie list} \cup 
+\underbrace{\{v_{ik}v_{jk} : v_iv_j \in E(G) \land k\in L_{v_i} \cap L_{v_j}\}}_\text{krawędzie kolorów}.\]
 
 Niech graf $G' = (V', E')$. 
 
-Prostrzymi słowami, każdy wierzchołek z $V(G)$ rozbijamy na dwa wierzchołki odpowiadające kolorom z listy (lub jeden, jeśli na liście znajduje się tylko jeden kolor) i tworzymy pomiędzy 
+Oznacza to, że każdy wierzchołek z $V(G)$ rozbijamy na dwa wierzchołki w $V'$ odpowiadające kolorom z listy (lub jeden, jeśli na liście znajduje się tylko jeden kolor) i tworzymy pomiędzy 
 nimi krawędź. Pozostałe krawędzie pomiędzy dwoma wierzchołkami
-$u$ i $v$ z $G'$ tworzymy wtedy kiedy wierzchołki 
+$u$ i $v$ z $G'$ tworzymy wtedy, kiedy wierzchołki 
 z których $u$ i $v$ zostały rozbite sąsiadują ze sobą
-w grafie $G$ oraz $u$ i $v$ odpowiadają temu samemu kolorowi.
+w grafie $G$ oraz $u$ i $v$ odpowiadają temu samemu kolorowi. 
 
-Wystarczy roztrzygnąć dwudzielność 
-grafu $G'$, przy pomocy algorytmu, który zastosowaliśmy w zadaniu 5 z tematu ,,Przeszukiwanie grafów''. Jeśli graf $G'$ jest dwudzielny
-to możemy pokolorować go listowo.
+Wystarczy rozstrzygnąć dwudzielność 
+grafu $G'$, przy pomocy algorytmu, który zastosowaliśmy w \hyperref[exc:bipart]{zadaniu 5 z tematu ,,Przeszukiwanie grafów''}. Jeśli graf $G'$ jest dwudzielny i każdy wierzchołek z $G$ ma swojego odpowiednika w \emph{tej samej} klasie dwudzielności $X$, to możemy pokolorować go listowo.

```

Comment:
```suggestion
grafu $G'$ przy pomocy algorytmu opisanego w \hyperref[exc:bipart]{zadaniu 5 z tematu ,,Przeszukiwanie grafów''}. Graf $G'$ jest dwudzielny i każdy wierzchołek z $G$ ma swojego odpowiednika w \emph{tej samej} klasie dwudzielności $X$ wtedy i tylko wtedy, gdy możemy pokolorować go listowo.

```

1. @kubazuch może link powinien być niebieski? :3
2. zmieniłem implikację na równoważność, bo to pokazujemy w kolejnych dwóch akapitach

Reply:
1. nie należy to do mojego issue
2. okej